CONTRIBUTING.md

```diff
@@ -73,15 +73,8 @@ Please ensure that the email address you use at Eclipse is the same as the "Publ
 
 ### Making the Commit
 
-When making the commit for the pull request it is vital that you "sign-off" on the commit using `git commit -s` option.
-Without this sign-off, your patch cannot be applied to the Winery repository because it will be rejected.
+In the case of multiple authors, please add `Also-by: Some Bodyelse <somebodyelse@nowhere.com>` for each additional author.
```

Comment:
I'd update this line to `Co-authored-by: Some Bodyelse <somebodyelse@nowhere.com>`  as this is also recognized by GtiHub and Eclipse also supports it.

Reply:
Fixed, and added it a separate block for easier copy-pasting.